etc/avocado/avocado.conf

```diff
@@ -38,6 +38,18 @@ utf8 =
 # Keep job temporary files after jobs (useful for avocado debugging)
 keep_tmp_files = False
 
+[remoter.behavior]
+# Reject unknown SSH host keys.
+# 'False' will leave you wide open to man-in-the-middle attacks!
+# 'True' will only work with RSA keys (due to a bug in Paramiko).
+# If using 'True', accept the remote host key fingerprint by using:
+#   $ ssh -oHostKeyAlgorithms='ssh-rsa' <host>
+reject_unknown_hosts = False
+# SSH layer to skip loading the user’s known-hosts file. Useful for avoiding
+# exceptions in situations where a 'known host' changing its host key is
+# actually valid (e.g. cloud servers such as EC2.)
+disable_known_hosts = False
```

Comment:
As you need to touch the selftest, it might be good to add:
```
# __Insecure__, ...
```
prefix to each of these options.